frontend/utilities/endpoints.ts

```diff
@@ -48,19 +50,23 @@ export default {
   MDM_PROFILES_AGGREGATE_STATUSES: `/${API_VERSION}/fleet/mdm/apple/profiles/summary`,
   MDM_APPLE_DISK_ENCRYPTION_AGGREGATE: `/${API_VERSION}/fleet/mdm/apple/filevault/summary`,
   MDM_APPLE_SSO: `/${API_VERSION}/fleet/mdm/sso`,
-  MDM_APPLE_EULA_FILE: (token: string) =>
-    `/${API_VERSION}/fleet/mdm/apple/setup/eula/${token}`,
   MDM_APPLE_ENROLLMENT_PROFILE: (token: string) =>
     `/api/mdm/apple/enroll?token=${token}`,
   MDM_BOOTSTRAP_PACKAGE_METADATA: (teamId: number) =>
     `/${API_VERSION}/fleet/mdm/apple/bootstrap/${teamId}/metadata`,
   MDM_BOOTSTRAP_PACKAGE: `/${API_VERSION}/fleet/mdm/apple/bootstrap`,
   MDM_BOOTSTRAP_PACKAGE_SUMMARY: `/${API_VERSION}/fleet/mdm/apple/bootstrap/summary`,
+  MDM_: `/${API_VERSION}/fleet/mdm/apple/setup/eula`,
```

Comment:
typo?

Reply:
Or remove because setup requires a token used in line 60

Reply:
yah meant to remove this. thanks for catching it.